src/olympia/abuse/views.py

```diff
@@ -9,18 +9,23 @@
 )
 from olympia.accounts.views import AccountViewSet
 from olympia.addons.views import AddonViewSet
-from olympia.api.throttling import GranularUserRateThrottle
+from olympia.api.throttling import GranularIPRateThrottle, GranularUserRateThrottle
 
 
 class AbuseThrottle(GranularUserRateThrottle):
```

Comment:
Would it make sense to rename this to `AbuseUserThrottle` to be consistent with the new `AbuseIPThrottle`?

Reply:
I agree - 93c84173fb548e4be310032a61d51d878a02c0f8